gulpfile.js

```diff
@@ -202,6 +203,21 @@ gulp.task('imagemin', function () {
     .pipe(gulp.dest('public/dist/img'));
 });
 
+// Copy local development environment config
+gulp.task('copy:localConfig', function () {
```

Comment:
The colon syntax is a remnant from Grunt, and I'd rather it not come over to Gulp as I think it's unclear. Can we try to rename these so they make sense? How about `copyLocalConfig`?